src/elements/header/header/readme.md

```diff
@@ -71,7 +71,7 @@ Finally, the following custom CSS has been added(\*). The result can be seen in
   }
 }
 
-@media screen and (width < 1024px) {
+@media screen and (width < 1023px) {
```

Comment:
as it uses "less than", I think 1024px would still be the correct value?

Reply:
Hmm, i just noticed the delta with the storybook's preview-head.html while looking for the spacer class, and then aligned it, but actually your point is correct.
I'll fix them both